docs/upgrade-to-v5.md

```diff
@@ -15,19 +15,28 @@ for nodejs and web platforms.
 
 > **TL;DR**: Web environments return Uint8Array; Node.js environments return Buffer
 
-For those that use the BSON library on Node.js, there is no change the BSON APIs will still return and accept instances of Node.js Buffer. Since we no longer depend on the Buffer web shim for compatibility with browsers, in non-Node.js environments a Uint8Array will be returned instead.
+For those that use the BSON library on Node.js, there is no change - the BSON APIs will still return and accept instances of Node.js Buffer. Since we no longer depend on the Buffer web shim for compatibility with browsers, in non-Node.js environments a Uint8Array will be returned instead.
 
 This allows the BSON library to be better at platform independence while keeping its behavior consistent cross platform. The Buffer shim served the library well but brought in more than was necessary for the concerns of the code here.
 
 ### `ObjectId.toString` / `UUID.toString` / `Binary.toString`
 
 > **TL;DR**: These `toString` methods only support the following encodings: 'hex', 'base64', 'utf8'
```

Comment:
if the 'utf8' encoding isn't supported on `ObjectId.toString` or `UUID.toString`, then this summary is incorrect

I actually think that what you currently have as a "tl;dr" can be simply the sub-heading for the corresponding set of changes. I think it would also make the table of contents easier to generate. In the case of this TL;DR, it can be removed entirely if you update L22 the way I am suggesting; then the paragraph underneath can elaborate on the specifics

Then the TL;DR of the section above (L16) can be promoted to a subheading, with both of these sections being under the title "Remove reliance on Node.js buffer", which would be the grouping heading.

So in this guide, the grouping headings will indicate the themes of the changes, and the subheadings in the groups will be the actual breaking change from the user's perspective. (What I mean is, "remove reliance on nodejs buffer" isn't the breaking change, it's the cause of the breaking changes whose summaries follow)

As for the 3rd example, I think we can group it under "Other" for now